src/x509.cc

```diff
@@ -45,6 +45,85 @@ std::string parse_args(const Nan::FunctionCallbackInfo<v8::Value>& info) {
   return *String::Utf8Value(info[0]->ToString());
 }
 
+
+
+NAN_METHOD(verify) {
+  Nan::HandleScope scope;
+  OpenSSL_add_all_algorithms();
```

Comment:
That's right, on my dev machine I have an earlier version:
```
openssl version
OpenSSL 1.0.2d 9 Jul 2015
```
So, what are we going to do? should I leave the line as it is?